app/src/main/res/values-en/strings.xml

```diff
@@ -974,9 +974,9 @@ If there are no signs along the whole street which apply for the highlighted sec
     <string name="quest_maxspeed_answer_noSign_urbanOrRural_description">Roads in built-up areas have a lower default speed limit than rural roads.</string>
     <string name="quest_maxspeed_answer_noSign_urbanOk">"Built-up area"</string>
     <string name="quest_maxspeed_answer_noSign_ruralOk">"Rural"</string>
-    <string name="quest_maxspeed_answer_noSign_info_zone">"If there is a sign like this at the main street intersection, you won’t find individual signs within the zone because the speed limit posted there applies to the whole zone."</string>
-    <string name="quest_maxspeed_answer_noSign_singleOrDualCarriageway_description">Are the roadways of this road here physically separated (i.e. through a barrier)? The default speed limit depends on whether this is the case or not.</string>
-    <string name="default_disabled_msg_maxspeed">This quest type is disabled by default because you are usually required to check the whole street for speed limit signs, not just the highlighted section. This can sometimes be quite the stretch just to solve a single quest.</string>
+    <string name="quest_maxspeed_answer_noSign_info_zone">"If there is a sign like this at the intersection with the main road, you may not find individual signs within the zone, as the speed limit posted there applies to the whole zone."</string>
+    <string name="quest_maxspeed_answer_noSign_singleOrDualCarriageway_description">Are opposite directions of traffic separated with a physical barrier? The default speed limit depends on whether this is the case.</string>
```

Comment:
It sounds like the original string’s “i.e.” was intended to clarify that a grassy median counts too, not just what a layperson might consider a barrier (like a guardrail or K-rail).

```suggestion
    <string name="quest_maxspeed_answer_noSign_singleOrDualCarriageway_description">Are opposite directions of traffic separated by a physical barrier or greenspace? The default speed limit depends on whether this is the case.</string>
```